fuse_constraints/include/fuse_constraints/absolute_constraint_impl.h

```diff
@@ -48,10 +48,11 @@ namespace fuse_constraints
 
 template<class Variable>
 AbsoluteConstraint<Variable>::AbsoluteConstraint(
+  const std::string& source,
   const Variable& variable,
   const fuse_core::VectorXd& mean,
   const fuse_core::MatrixXd& covariance) :
-    fuse_core::Constraint{variable.uuid()},
+    fuse_core::Constraint(source, {variable.uuid()}),  // NOLINT(whitespace/braces)
```

Comment:
I think this should also work: `fuse_core::Constraint{source, variable.uuid()},` (same in other instances below)

Reply:
That particular incantation results in:
```
error: no matching function for call to ‘std::vector<boost::uuids::uuid>::vector(boost::uuids::uuid&)’
```
This does work, but I'm not sure it has any advantages over the constructor `()` version:
```
fuse_core::Constraint{source, {variable.uuid()}}
```
I could do something with variadic templates:
https://stackoverflow.com/questions/25721869/initializer-list-combined-with-other-parameters
```
public:
  template <class ...Args>
  Constraint(const std::string& source, Args... args) : Constraint(source, {args...}) {}
private:
  Constraint(const std::string& source, std::initializer_list<UUID> variable_uuid_list) :
    source_(source),
    variables_(variable_uuid_list)
  {}
```
but I'm not sure it is worth the trouble.

Reply:
Fair enough. I don't think it's worth the trouble, and I didn't realized there's an `std::initializer_list` for the UUIDs.